### ISSUE TITLE
About size of input images for training

### ISSUE THREAD

**Issue by bao-O** [qjadud1994/CRNN-Keras#47]
I have two question:
- The first, I have some real traffic images that I want to use for training model. But when I crop it to get only the plate, it maybe small ( ~ 50px). What size of the input images should be? And does it effect significantly to the accuracy of the model after training?
- The second, I want to train model for recognize both two and one layer plate. So how many image of each class should be prepared for a accuracy model?
Thanks in advance!

**Comment by ZdsAlpha** [comment]
Looking through code it looks like height of image is 64 pixels and you can have arbitrary width as it pads/crops the rest of image that does not fit.

**Comment by qjadud1994** [comment]
1. Here, 128x64 is used. Customize the image size to suit you.
2. The number required depends on the characteristic of the training data.
Also, due to the structure of the CRNN, two-line license plates may not be recognized well.
